docs/usage/shoot_overlay_network.md

```diff
@@ -0,0 +1,77 @@
+# Enable / disable overlay network for shoots with Calico
+Gardener can be used with or without the overlay network. From now on, the default configuration of shoot clusters is without overlay network.
+
+## Understanding overlay network
+The Overlay networking permits the routing of packets between multiples pods located on multiple nodes, even is the pod and the node network are not the same.
+
+This is done through the encapsulation of pod packets in the node network so that the routing can be done as usual. We use `ipip` encapsulation with calico in case the overlay network is enabled. This (simply put) sends an IP packet as workload in another IP packet.
+
+![](./images/Overlay-Network.drawio.png)
+
+In order to simplify the troubleshooting of problems and reducing the latency of packets travelling between nodes, the no overlay network is now enabled by default for all new clusters
```

Comment:
Similar to the above comment - it is not clear what is "now" and it is not enabled by default.